Core/Graphics/Source/Graphics.cpp

```diff
@@ -249,6 +260,26 @@ namespace Babylon
         Callback.SetDiagnosticOutput(std::move(outputFunction));
     }
 
+    float Graphics::Impl::GetHardwareScalingLevel()
+    {
+        std::scoped_lock lock(m_resolution.Mutex);
+        return m_resolution.hardwareScalingLevel;
+    }
+
+    void Graphics::Impl::SetHardwareScalingLevel(float level)
+    {
+        if (level == 0)
```

Comment:
I guess we should check for <= 0 since it is an unsigned float?

Reply:
<= FLT_EPSILON . Never compare floats with literal values.

Reply:
Thanks, both these changes make sense.

Reply:
`std::numeric_limits<float>::epsilon()` if you want the C++ way.

Reply:
Looks like `FLT_EPSILON` is not supported on some platforms anyway.

Reply:
It should be. Probably just need to include the header. I would recommend using the C++ one though.

Reply:
Ok, decided to go with the c++ style for this, thanks for the suggestions.

Reply:
Include <limits>

Reply:
Added the include.